pegasus/sites.v3/code.org/public/athletes.haml

```diff
@@ -17,30 +17,24 @@ social:
   "twitter:url": "https://code.org/athletes"
   "twitter:title": "<%= I18n.t(:sports_og_title) %>"
   "twitter:description": "<%= I18n.t(:sports_og_description) %>"
-  "twitter:image:src" : "<%= 'https://' + request.host + '/images/social-media/' + (PageMode.get(request) == 'feature-athletes' ? 'athletes-thumbnail.jpg' : 'sports.jpg') %>"
-  "twitter:player": "<%= 'https://youtube.com/embed/' + (PageMode.get(request) == 'feature-athletes' ? 'ip051U7Rvds' : 'QvyTEx1wyOY') %>"
+  "twitter:image:src" : "<%= 'https://' + request.host + '/images/social-media/athletes-thumbnail.jpg' %>"
+  "twitter:player": "<%= 'https://youtube.com/embed/ip051U7Rvds' %>"
   "twitter:player:width": "640"
   "twitter:player:height": "360"
 video_player: true
 ---
-
-- page_mode     = PageMode.get(request)
-- yt_code       = page_mode == 'feature-athletes' ? 'ip051U7Rvds' : 'QvyTEx1wyOY'
-- download_link = page_mode == 'feature-athletes' ? '//videos.code.org/social/athletes-hour-of-code.mp4' : '//videos.code.org/social/cs-is-everything.mp4'
-
 %link{href: "/css/sports.css", rel: "stylesheet"}
 
 %div{style: "clear:both"}
 
 .tutorial-video-container
   %h2= I18n.t(:sports_video_title)
-  - url = "https://www.youtube.com/watch?v=#{yt_code}"
+  - url = "https://www.youtube.com/watch?v=ip051U7Rvds"
   - facebook = {u: url}
   - twitter = {url: url, related: 'codeorg', text: "#{I18n.t(:sports_og_description)} @codeorg"}
-  =view :display_video_thumbnail, id: 'athletes_video', video_code: yt_code, play_button: 'center', facebook: facebook, twitter: twitter, download_path: download_link, letterbox: 'false'
-  - if page_mode == "feature-athletes"
-    %a{href: "#featured-athletes"}
-      = I18n.t(:sports_featured_athletes)
+  =view :display_video_thumbnail, id: 'athletes_video', video_code: 'ip051U7Rvds', play_button: 'center', facebook: facebook, twitter: twitter, download_path: '//videos.code.org/social/athletes-hour-of-code.mp4', letterbox: 'false'
```

Comment:
Correct, it's the same name but the file was updated in aws. Just checked and downloaded now to confirm it has Kobe.